lib/client-assets.php

```diff
@@ -149,6 +149,25 @@ function gutenberg_override_translation_file( $file, $handle ) {
 }
 add_filter( 'load_script_translation_file', 'gutenberg_override_translation_file', 10, 2 );
 
+/**
+ * Filters the default labels for common post types to change the case style
+ * from capitalized (e.g. "Featured Image") to sentence-style (e.g. "Featured
+ * image").
+ *
+ * See: https://github.com/WordPress/gutenberg/pull/18758
+ *
+ * @param object Object with all the labels as member variables.
+ *
+ * @return object Object with all the labels, including overridden ones.
+ */
+function gutenberg_override_posttype_labels( $labels ) {
```

Comment:
Why do we need this function in Gutenberg?

Reply:
@Soean the problem is that our labels come from core so we can't have sentence case without. I will loop @mcsf in here as they worked on this solution to help.

Reply:
Then we have to change it in Core :-)

Reply:
@Soean, thanks for that watchful eye. :) Indeed, we need to change it in core.

c7f1e892 was a drive-by commit to unblock Tammie. The commit message itself did mention the need to address this in core, but commit messages aren't very visible here. This all came from [this thread in Slack](https://wordpress.slack.com/archives/C02QB2JS7/p1576067206315600), which also acknowledged the need to address the root cause.

The question that this commit raised is: in the interest of landing something consistent here in #18758, _can we_ add this filter for the plugin, provided that we also open the appropriate Trac tickets?

Reply:
Thanks for clarification. 
If a post type has a different label than "Featured image" for example "Book cover", then the filter overrides the value? This could be a problem.

Reply:
>  If a post type has a different label than "Featured image" for example "Book cover", then the filter overrides the value? This could be a problem.

This change is only targeting `post` and `page` post types. Do you think that, even then, we should guard against potential customisations? Something like:

```php
if ( __( 'Featured Image' ) == $labels->featured_image ) {
  $labels->featured_image = __( 'Featured image', 'gutenberg' );
}
return $labels;
```

Reply:
@Soean can you confirm if what @mcsf has done is ok for now? I will happily make a trac ticket, but it would be good to get a start on things in Gutenberg here.

Reply:
Yes, now it's translateable and it works for post und page post type. 
So we can merge it and link it in the trac ticket, so we can remove it after merging it in core.

Reply:
Hi @mcsf, it seems this change needs to be backported to core. Would you be able to submit a core patch?

Reply:
Will do, @jorgefilipecosta.

Reply:
→ https://core.trac.wordpress.org/ticket/49371

Reply:
Done